sqlserver-delta-plugins/src/e2e-test/java/io.cdap.plugin/MssqlClient.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2023.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin;
+
+import io.cdap.e2e.utils.PluginPropertyUtils;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.TimeZone;
+import java.util.concurrent.TimeUnit;
+
+/**
+ *  Mssql client.
+ */
+public class MssqlClient {
+
+    static Connection getMssqlConnection() throws SQLException, ClassNotFoundException {
+        TimeZone timezone = TimeZone.getTimeZone("UTC");
+        TimeZone.setDefault(timezone);
+        Class.forName("com.microsoft.sqlserver.jdbc.SQLServerDriver");
+        String databaseName = PluginPropertyUtils.pluginProp("mssqlDatabaseName");
+        return DriverManager.getConnection("jdbc:sqlserver://" + System.getenv("MSSQL_HOST")
+                        + ":" + System.getenv("MSSQL_PORT") + ";databaseName=" + databaseName,
+                System.getenv("MSSQL_USERNAME"), System.getenv("MSSQL_PASSWORD"));
+
+    }
+
+    public static void createTable(String table, String schema, String datatypeColumns)
+            throws SQLException, ClassNotFoundException {
+        try (Connection connect = getMssqlConnection(); Statement statement = connect.createStatement()) {
+            String createTableQuery = "CREATE TABLE " + schema + "." + table + datatypeColumns;
+            statement.executeUpdate(createTableQuery);
+            String enableCDCSql = "EXEC sys.sp_cdc_enable_table @source_schema = " + schema + " , " +
```

Comment:
You can add a comment here saying that we are enabling table level CDC here.

Reply:
Added the comment accordingly.